src/notebook-jobs-panel.tsx

```diff
@@ -59,11 +67,120 @@ export class NotebookJobsPanel extends VDomRenderer<JobsModel> {
     this._translator = options.translator;
     this._trans = this._translator.load('jupyterlab');
     this._advancedOptions = options.advancedOptions;
+    this._last_input_drop_target = null;
 
     this.node.setAttribute('role', 'region');
     this.node.setAttribute('aria-label', trans.__('Notebook Jobs'));
   }
 
+  removeDraghoverClass = (event: Event): void => {
```

Comment:
```suggestion
  removeDragHoverClass = (event: Event): void => {
```

Let's treat "drag" and "hover" as separate words for the purpose of camelCase.